gym_sts/envs/base.py

```diff
@@ -350,7 +346,13 @@ def step(self, action_id: int) -> Tuple[dict, float, bool, dict]:
         action = ACTIONS[action_id]
         is_valid = validate(action, prev_obs)
 
-        obs = self.communicator._manual_command(action.to_command())
+        try:
+            obs = self.communicator._manual_command(action.to_command())
+        except exceptions.StSError as e:
+            print(prev_obs)
+            print(action_id)
+            self.screenshot("error.png")
```

Comment:
Perhaps add a timestamp to the error? I think the screenshot tool is a bit unreliable when the file already exists under the same name.